lambda-notes/src/App.js

```diff
@@ -9,10 +12,23 @@ const MainContainer = styled.div`
   margin: 0 auto;
 `
 
+const SidebarStyled = styled(Col)`
+  background-color: #D8D8D8;
+`
+const ContentStyled = styled(Col)`
+  background-color: #F3F3F3;
+`
 class App extends Component {
   render() {
     return (
-      <MainContainer>Hello</MainContainer>
+      <MainContainer>
```

Comment:
Enjoyed seeing `MainContainer` get filled out with `SidebarStyled` and `ContentStyled` using **Reactstrap.**